ydb/core/tx/datashard/local_kmeans.cpp

```diff
@@ -544,10 +549,10 @@ class TLocalKMeansScan final: public TLocalKMeansScanBase, private TCalculation<
         return true;
     }
 
-    EScan FeedSample(const TRow& row)
+    EScan FeedSample(TArrayRef<const TCell> row)
```

Comment:
FeedSample is duplicated between LocalKMeans and PrefixKMeans, I forgot about this when copied local_kmeans to write prefix_kmeans.

Reply:
https://github.com/ydb-platform/ydb/issues/16349